tools/ctl/linux/aws-otel-collector-ctl.sh

```diff
@@ -70,6 +70,8 @@ aoc_config_local_uri() {
 
     if [ -n "$config" ] && [ -f "$config" ]; then
         cp "$config" $CONFDIR/config.yaml
+    elif [ "$config" = "$CONFDIR/config.yaml" ]; then
```

Comment:
This check needs to be nested within the first conditional and not in an `elif`.